tutorials/editor/project_settings.rst

```diff
@@ -5,12 +5,21 @@
 Project Settings
 ================
 
-This page explains how to use the Project Settings window. If you would like to access and modify project settings via code, see :ref:`ProjectSettings <class_ProjectSettings>`.
+.. warning:: This page is a work-in-progress.
```

Comment:
We should not have notes of this form. We should bring the page to a correct state, even if it is very short, and not directly comment on future changes within the text of a page.